packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java

```diff
@@ -72,6 +72,7 @@ public void onMethodCall(MethodCall call, Result result) {
 
   private void startService(JSONArray arguments) throws JSONException {
     long callbackHandle = arguments.getLong(0);
+    AlarmService.setCallbackDispatcher(mContext, callbackHandle);
```

Comment:
Can these two be folded together into one call?

Reply:
They could be, but `AlarmService.startAlarmService` is also called from `AlarmService.onCreate` which retrieves this callback handle anyway, so having them separate avoids setting the callback dispatcher handle when it hasn't changed.

Reply:
ack.